updating/updating-restricted-network-cluster/uninstalling-osus.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-osus"]
+= Uninstalling the OpenShift Update Service from a cluster
+include::_attributes/common-attributes.adoc[]
+:context: uninstalling-osus
+
+toc::[]
+
+If you have a local copy of the OpenShift Update Service on your cluster, you can use the following procedures to uninstall it from the cluster.
```

Comment:
It might be helpful to specify in the assembly intro that the user has to do both types of procedures (delete and then uninstall)